app/components/stacks-by-organization/component.js

```diff
@@ -0,0 +1,24 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  organizationGroups: function() {
+    var groups = Ember.A([]);
+
+    this.get('stacks').forEach(function(stack) {
+      var currentOrganizationId = stack.get('_data.links.organization');
+      var currentOrganization = stack.get('organization');
+
+      if(!groups.findBy('organizationId', currentOrganizationId)) {
+        groups.addObject(Ember.Object.create({
+          organizationId: currentOrganizationId,
+          organization: currentOrganization,
+          stacks: Ember.A([])
+        }));
+      }
+
+      groups.findBy('organizationId', currentOrganizationId).get('stacks').addObject(stack);
+    });
```

Comment:
Oh, also, is it possible to read the `self` link from `organization` instead of peeking into `_data`? That is definitely a private API and not very stable to build upon.

@bantic is there a good way to read an arbitrary link URL or object via meta or something?